packages/docs/src/routes/docs/(qwikcity)/project-structure/index.mdx

```diff
@@ -43,7 +43,7 @@ The `src/routes/` directory is a special directory for Qwik City since it's the
 - `src/routes/index.tsx` is the homepage of your app.
 - `src/routes/layout.tsx` is the root layout of your app, all pages will be rendered inside this layout.
 
-> Refer to the [Routing](/qwikcity/routing) section for more information.
+> Refer to the [Routing](/docs/(qwikcity)/routing) section for more information.
```

Comment:
thanks! the (qwikcity) should be invisible to the router, so I guess it  should be `/docs/routing`

Reply:
Yes, you're right. I made a mistake with the routing, but now it's correct. @shairez 👀

Reply:
thanks!